apps/app/package.json

```diff
@@ -43,7 +42,7 @@
     "styled-components": "5.3.5",
     "styled-normalize": "^8.0.7",
     "tabletop": "1.6.3",
-    "wrangler": "^2.0.24"
+    "wrangler": "2.14.0"
```

Comment:
We can also remove this.

Reply:
I think this is necessary because it makes me interact with cloudflare pages API in an easy way. The other one was conflicting with it since this one is a newer version so I deleted the other one only.